thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/DetectionResource.java

```diff
@@ -540,6 +541,41 @@ public Response createUserAnomaly(
     return Response.ok(anomaly.getId()).build();
   }
 
+  /**
+   * Toggle active/inactive for given detection
+   *
+   * @param detectionId detection config id (must exist)
+   */
+  @PUT
+  @Path("/activation/{id}")
+  @ApiOperation("Make detection active or inactive, given id")
+  public Response toggleActivation(
+      @PathParam("id") long detectionId) throws Exception {
+    Map<String, String> responseMessage = new HashMap<>();
+    Boolean toggledTo = false;
```

Comment:
1. Why use Boolean instead of "boolean"?
It requires box/unbox.
2. Default value is false so no need to set explicitly.
3. This is only a temporarily used variable. Declare a variable to its minimum scope.  
No need to declare  before  the "try" block.